pack/fhv/fhvc.json

```diff
@@ -118,9 +118,349 @@
+    "quantity": 1,
+    "stage": 2,
+    "text": "<b>Forced</b> – When your turn ends, if you are at a [[Bog]] location: Place 1 damage on your location, as a sinkhole (2 sinkholes instead if there are 1 or 2 investigators in the game).\n<b>Forced</b> – If a location has 3 or more sinkholes on it: Remove all sinkholes from it and flip it over.\n<b>Objective</b> – If the Thing in the Depths is defeated, <b>(→R3)</b>.",
+    "type_code": "agenda"
+  },
+  {
+    "back_flavor": "You come to a round still pond on the northeastern shore of the bog. Atop a mound of wildflowers is a silvery alkaloid blossom very much like the sample that drew Dr. Marquez to Hemlock Isle. When you approach, the blossom swells to release a cloud of spores, then closes. The scent of the shoes is planet, almost like cloves, but grows bitter the longer it lingers. Suddenly the mound of flowers moves. A large reptilian head rises out of the water and looks at you with intelligent eyes before swiftly swimming away.",
+    "back_name": "The Wise Mound",
+    "back_text": "Spawn the set-aside Chelydran Hybrid enemy at the North Shore location. Choose a location adjacent to Chelydran Hybrid's location and spawn 1 of the set-aside Grasping Tendril enemies at that location.\n Shuffle each other set-aside Grasping Tendril enemy into the encounter deck along with the discard pile.",
+    "code": "10591",
+    "clues": 3,
+    "double_sided": true,
+    "encounter_code": "the_thing_in_the_depths",
+    "encounter_position": 4,
+    "faction_code": "mythos",
+    "flavor": "The bog is supposedly the origin of the sample that drew Dr. Marquez to the island in the first place. You have your work cut out for you.",
+    "illustrator": "Rob Laskey",
+    "name": "A Botanical Survey",
+    "pack_code": "fhvc",
+    "position": 91,
+    "quantity": 1,
+    "stage": 1,
+    "text": "<b>Objective</b> – At the end of the round, investigators at the North Shore location may spend the requisite number of clues, as a group, to advance. <i>See the Location Placement diagram on page 28 of the campaign guide for the North Short location.)</i>",
+    "type_code": "act"
+  },
+  {
+    "back_flavor": "You corner the reptilian creature in a shining pool, where it sinks into the mud, clearly exhausted. As it starts munching on a tangle of flowers, you take several sketches and notes of the thing's sprouting shell. The alkaloid blossom on its back seems to pulse. Producing a pair of shears, you prepare to take a sample.",
+    "back_name": "Strange Anatomy",
+    "back_text": "If it is <b>Day 1</b>, read the following. Otherwise, <b>(→R1)</b>.<blockquote>\"Stop!\" Dr. Marquez shouts. \"That flower is parasitic. It is now part of the creature's anatomy. There's no telling how deeply it's embedded.\" She points to the flower's thick translucent stem: vicious white fluid pumps through it like an artery. \"If the flower dies, its host dies too.\"</blockquote>\n<b>(→R2)</b> <i>(page 30)</i>",
```

Comment:
```suggestion
    "back_text": "<blockquote><i>You corner the reptilian creature in a shining pool, where it sinks into the mud, clearly exhausted. As it starts munching on a tangle of flowers, you take several sketches and notes of the thing's sprouting shell. The alkaloid blossom on its back seems to pulse. Producing a pair of shears, you prepare to take a sample.</i></blockquote>\n<hr>If it is <b>Day 1</b>, read the following. Otherwise, <b>(→R1)</b>.\n<blockquote><i>\"Stop!\" Dr. Marquez shouts. \"That flower is parasitic. It is now part of the creature's anatomy. There's no telling how deeply it's embedded.\" She points to the flower's thick translucent stem: vicious white fluid pumps through it like an artery. \"If the flower dies, its host dies too.\"</i></blockquote>\n<b>(→R2)</b> <i>(page 30)</i>.",
```